.github/workflows/no-warns.yaml

```diff
@@ -0,0 +1,23 @@
+name: No warnings
+
+on:
+  push:
+    branches:
+      - '*'
+      - '!main'
+  pull_request:
+    branches: [ main ]
+  workflow_dispatch:
+
+env:
+  PACKAGE_NAME: aws-crt-kotlin
+
+jobs:
+  ktlint:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout sources
+        uses: actions/checkout@v2
+      - name: No warnings on ${{ env.PACKAGE_NAME }}
+        run: |
+          ./gradlew -PkotlinWarningsAsErrors=true
```

Comment:
fix

This doesn't run a compile task by default. This will not do anything other than run the configure step of gradle and printout an [informational message](https://github.com/awslabs/aws-crt-kotlin/runs/3885464857?check_suite_focus=true).

I think maybe we should just integrate it [here](https://github.com/awslabs/aws-crt-kotlin/blob/main/.github/workflows/ci.yaml#L34) into normal CI build checks by creating a `local.properties` file in the GH action _before_ running `builder.pyz build`

Reply:
Good point, will move into CI instead.